### PR TITLE
Invoke tasks support for both windows and posix systems

### PR DESCRIPTION
**Pull request by patjouk**
I forgot that windows and *Nix systems handle venv differently.
Not sure if it will work on the first try, but I don't have a windows system to test it. @Pomax can you check and edit this PR if necessary? Works fine on MacOS.

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -5,48 +5,61 @@
...
+os.environ.pop("__PYVENV_LAUNCHER__", None)
 
 ROOT = os.path.dirname(os.path.realpath(__file__))
-
-# Python commands's outputs are not rendering properly. Setting pty for *Nix system and
-# "PYTHONUNBUFFERED" env var for Windows at True.
-if platform == 'win32':
-    PLATFORM_ARG = dict(env={'PYTHONUNBUFFERED': 'True'})
+VENV_DIR = "pulseenv"
+
```
Comment (Pomax) [inline tasks.py:12]: oops

```suggestion
```

File: tasks.py [hunk 2]
```diff
@@ -85,8 +98,8 @@ def catch_up(ctx):
 def new_db(ctx):
     """Create a new database with fake data"""
     print("* Reset the database")
-    ctx.run("dropdb --if-exists -h localhost pulse -U postgres")
-    ctx.run("createdb -h localhost pulse -U postgres")
+    ctx.run("dropdb --if-exists -h localhost -U postgres pulse")
```
Comment (Pomax) [inline tasks.py:101]: got errors on Windows due to this ordering, which is weird, but also an easy fix

File: tasks.py [hunk 3]
```diff
@@ -5,48 +5,59 @@
 
 # Workaround for homebrew installation of Python (https://bugs.python.org/issue22490)
 import os
-os.environ.pop('__PYVENV_LAUNCHER__', None)
+os.environ.pop("__PYVENV_LAUNCHER__", None)
```
Comment (Pomax) [inline tasks.py:8]: we were mixing `'` and `"` so a bunch of these changes are just cosmetic

Comment (patjouk) [reply tasks.py:8]: Thanks!

**Comment by patjouk** [comment]
Reverting a change I made: we only need the `python`/`python3` for creating the venv. After that, we're calling `python` inside the venv.

**Comment by Pomax** [comment]
A, good find - still works on my end!

**Pull request merged**